### PR TITLE
Tuned v2 merge

### PR DESCRIPTION
**Pull request by satyaog**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #[69, 68]

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
- [x] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

- Improved testing logic to consider various test types
- Some symptom properties are converted to functions for better readability
- Cold and Flu transmission and symptoms are modeled
- Calculation of R is improved

- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by satyaog** [comment]
@pg2455 , the code should be fixed now. You can validate the last commit then approve this PR so it can be merged

**Review by pg2455 — APPROVED** [review]
This PR is a merging of several branches - symptoms_cold_flu, tuned_v2, improve_R. 

We (me and @satyaog) have resolved the conflicts and went over the code together.

File: base.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (obilaniu) [inline base.py:1]: Strange, absolutely necessary? Shifts all line numbers by 1.

File: base.py [hunk 2]
```diff
@@ -44,7 +45,7 @@ def time_of_day(self):
 
 class City(object):
 
-    def __init__(self, env, n_people, rng, x_range, y_range, start_time, init_percent_sick, Human, sim_days):
```
Comment (obilaniu) [inline base.py:47]: It's not the business of the City to duplicately know how long to run itself (any any units, whether days, seconds, ticks, ...)

The Env (already passed in as first argument) ought to know though.

**Comment by obilaniu** [comment]
Although this is big, I'm hoping it's the last great earthquake before we move in to stabilizing it. It has my blessing to merge in.

**Pull request merged**